docker/Dockerfile.py3.base

```diff
@@ -0,0 +1,2 @@
+ARG pythonversion
```

Comment:
If I understand Pike's intentions correctly, this argument should land in `docker/Dockerfile.base` and `Dockerfile.py3.base` should be removed.

Reply:
Ok